### PR TITLE
Update Lavaplayer usage

### PR DESCRIPTION
**Pull request by szymonwilczek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,4 +247,24 @@ dependencies {
 
 ### Usage
 
-// TODO
\ No newline at end of file
+Create new `Lyrics Manager`:
```
Comment (topi314) [inline README.md:250]:
```suggestion
Create new `LyricsManager`:
```

File: README.md [hunk 2]
```diff
@@ -247,4 +247,24 @@ dependencies {
...
+```java
+var lyricsManager = new LyricsManager();
+```
```
Comment (topi314) [inline README.md:253]: also mention the need to register sources here

Comment (szymonwilczek) [reply README.md:253]: A quick

> Remember to register the sources

or a whole example?

Comment (szymonwilczek) [reply README.md:253]: or a comment insde of the `java block`?

Comment (topi314) [reply README.md:253]: a comment with some pseudo code like this should be fine
```suggestion
lyricsManager.registerLyricsManager(sourcer)
```

**Review by topi314 — APPROVED** [review]

**Comment by topi314** [comment]
thanks

**Pull request merged**